logstash.ml

```diff
@@ -265,7 +265,7 @@ let setup_error_log () =
     chain_hook level facil s
   end
 
-let lifetime ?(extra="") ~events ~version () =
+let lifetime ?(extra="") ?(start=Time.now ()) ~events ~version () =
```

Comment:
Time.now is not useful - better default to Action.uptime#start (or new toplevel value)
I was thinking no argument is needed but actually indeed some people may want to record uptime of forked processes precisely

Reply:
`start` is not exposed directly but I did the equivalent using `get_state`

Reply:
regarding forked processes, the exit event is not logged, which makes me think `lifetime` is not really used for them, so the argument might indeed not be needed

Reply:
I meant when lifetime is called in a forked process already, Action.uptime will still hold the parent's start time. But now I looked at the code more attentively and Time.now was correct, my comment was wrong, and no need for start argument at all actually :]